splunk-quarkus/init_ephemeral_cluster.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env
+
+# See step 7 of this link: https://clouddot.pages.redhat.com/docs/dev/getting-started/ephemeral/onboarding.html
+echo "Logging into Openshift cluster"
+sh ~/ephemeral-login.sh
+echo "Checking for reserved namespace"
+export NAMESPACE=$(bonfire namespace list --mine | grep "ephemeral" | awk '{print $1"|"$2}' | grep '|true' | awk -F'|' '{print $1}' | head -n1) # we use the first one available
+
+if [[ $NAMESPACE == *"ephemeral-"* ]]; then
+    echo "Namespace $NAMESPACE reserved, extending for 8 hours"    
+    bonfire namespace extend $NAMESPACE -d '8h0m' 
+else
+    echo "Reserving namespace for 8 hours"
+    export NAMESPACE=$(bonfire namespace reserve -d '8h0m')
+    echo "Namespace $NAMESPACE reserved"
+fi
+
+echo "Using $NAMESPACE as default oc project"
+oc project $NAMESPACE
+
+echo "Deploying apps to $NAMESPACE"
+bonfire deploy advisor drift gateway insights-ephemeral host-inventory notifications -n $NAMESPACE -t 1200 --source=appsre
```

Comment:
JFYI, sometimes drift takes resources that could hit limits of ephemeral env. I usually deploy only advisor as event producer for testing and skip drift.